### DISCUSSION TITLE
Switching theme gives error!

### DISCUSSION THREAD

**Discussion by Ovindu1u** [NvChad/NvChad#2530] [Q&A] [answered]
`E5108: Error executing lua: /home/ovindu/.local/share/nvim/lazy/ui/lua/nvchad/ut
ils.lua:27: attempt to index local 'file' (a nil value)
stack traceback:
        /home/ovindu/.local/share/nvim/lazy/ui/lua/nvchad/utils.lua:27: in funct
ion 'replace_word'
        .../share/nvim/lazy/ui/lua/telescope/_extensions/themes.lua:80: in funct
ion 'run_replace_or_original'
        ...re/nvim/lazy/telescope.nvim/lua/telescope/actions/mt.lua:65: in funct
ion 'key_func'
        ...hare/nvim/lazy/telescope.nvim/lua/telescope/mappings.lua:257: in func
tion <...hare/nvim/lazy/telescope.nvim/lua/telescope/mappings.lua:256>
`

also saving with :w switches back to the default theme

**Comment by siduck** [comment]
@Ovindu1u make sure chadrc exists

**Comment by Pix3lexe** [comment, marked as answer]
Did you find a solution? When I try to switch themes I get the error
```
E5108: Error executing lua. ....local/share/nvim/lazy/ui/lua/nvchad/themes/mappings.lua:23: attempt to index field 'base46' (a nil value)
stack traceback:
....local/share/nvim/lazy/ui/lua/nvchad/themes/mappings.lua:23: in function <....local/share/nvim/lazy/ui/lua/nvchad/themes/mappings.lua:19>
```

This is my `chadrc.lua`. Maybe is there an error with that? Also `:w` resets the theme if I'm in `.config/nvim`
![image](https://github.com/user-attachments/assets/10587e99-5670-4163-9f8f-f0a7dad03d0a)


EDIT: It was the chadrc.lua file. They updated the format and now it is `M.base46` intead of `M.ui`
